charts/flagsmith/README.md

```diff
@@ -1,10 +1,127 @@
 # Flagsmith Helm chart
 
-## Requirements
+## Installation
+
+```bash
+helm repo add flagsmith https://flagsmith.github.io/flagsmith-charts/
+helm install flagsmith flagsmith/flagsmith
+```
+
+This will install using default options. (Pods may error and restart
+while waiting for other pods to become ready.).
+
+### Ingress configuration
+
+The above will start pods in the cluster, but to gain web access to
+Flagsmith, do one of the following:
+
+#### In a cluster that has an ingress controller
+
+Set the following values for flagsmith, with changes as needed to
+accommodate your ingress controller, and any associated DNS
+changes. Also, set the `API_URL` env-var such that the URL is
+reachable from a browser accessing the frontend.
+
+Eg:
+
+```yaml
+ingress:
+  frontend:
+    enabled: true
+    hosts:
+      - host: flagsmith.[MYDOMAIN]
+        paths:
+          - /
+  api:
+    enabled: true
+    hosts:
+      - host: flagsmith.[MYDOMAIN]
+        paths:
+          - /api/
+
+frontend:
+  env:
+    - name: API_URL
+      value: 'https://flagsmith.[MYDOMAIN]/api/v1/'
+```
+
+Then access, once any out-of-cluster DNS or CDN changes have been
```

Comment:
Then, once rather than "Then access, once"?

Reply:
Agreed and fixed.